docs/rholang/rholangtut-0.3.md

```diff
@@ -108,44 +115,50 @@ Note the deep layers of callback. Rholang was designed to make concurrent comput
 
 ## Iteration and matching
 
-In the code below, we show an example of iterating through a linked list implemented as nested [head, tail] pairs.
-
-     1 new chan, loop, iCh in {
-     2   contract loop(@list, @acc, return) = {
-     3     match list { 
-     4       [hd, tl] => {
-     5         for(@i <- iCh) {
-     6           iCh!(i + 1) |
-     7           match [hd == i, acc] {
-     8             [true, true] => { loop!(tl, true, *return) }
-     9             _ => { loop!(tl, false, *return) }
-    10           }
-    11         }
-    12       }
-    13       _ => { return!(acc) }
-    14     }
-    15   } |
-    16   iCh!(1) |
-    17   loop!([1, [2, [3, [4, []]]]], true, *chan)
-    18 }
+In the code below, we show an example of iterating through a list.
+
+     1	new iterate in {
```

Comment:
Does this example work for you @pyrocto? I can't seem to get this to run as expected

Reply:
Looks like there's a bug in the interpreter that this will have to wait on:
https://rchain.atlassian.net/browse/RHOL-355

Reply:
Heh, just filed the same thing.  Deleted 355